docs/data-management-on-carbondata.md

```diff
@@ -122,6 +122,45 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
      TBLPROPERTIES ('streaming'='true')
      ```
 
+  - **Local Dictionary Configuration**
+  
+  Local Dictionary is generated only for no-dictionary string/varchar datatype columns. It helps in:
+  1. Getting more compression on dimension columns with less cardinality.
+  2. Filter queries and full scan queries on No-dictionary columns with local dictionary will be faster as filter will be done on encoded data.
+  3. Reducing the store size and memory footprint as only unique values will be stored as part of local dictionary and corresponding data will be stored as encoded data.
+
+       By default, Local Dictionary will be enabled and generated for all no-dictionary string/varchar datatype columns.
```

Comment:
Convert this into table

| Properties | Default Value | Description |

The **description** should satisfy the following points:
a.	What does this parameter do?
b.	In what scenario the user needs to configure this parameter?
c.	Are there any benefits in configuring this parameter?
d.	What is the default value?
e.	What is the value range if any?
f.	Are there any limitations?
g.	Any key information to be highlighted?

Reply:
By default, Local Dictionary will be enabled and generated for all no-dictionary string/varchar datatype columns. ----- Is the data loading performance ok?

Reply:
Data Loading Performance is affected only by 8%. Test with 3.5 billion records (103 columns)